spartan/terraform/deploy-release/deploy.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+RELEASE_NAME="rough-rhino"
```

Comment:
this needs to be insync with the tf vars file, probably worth just writing this as a TF_VAR_RELEASE_NAME and referencing that below